runtime/vkext.cpp

```diff
@@ -332,7 +332,9 @@ void write_char_utf8(int c) {
     write_buff_char_3((char)(0xe0 + (c >> 12)), (char)(0x80 + ((c >> 6) & 63)), (char)(0x80 + (c & 63)));
     return;
   }
-  if (c >= 0x1f000 && c <= 0x1f9ff) {
+  // todo I don't know, why this condition starts from 0x1f000
+  // maybe, it's a good idea to add all unicode blocks https://unicode-table.com/en/blocks/ or just start from 0x10000
+  if (c >= 0x1f000 && c <= 0x1ffff) {
```

Comment:
should we fix `write_char_utf8_no_escape` as well?

Reply:
Not sure: it has another range